Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -477,6 +477,56 @@ package actor SkipUnless {
     }
   }
 
+  package static func canSwiftPMCompileForIOS(
+    file: StaticString = #filePath,
+    line: UInt = #line
+  ) async throws {
+    struct NoSwiftInToolchain: Error {}
+
+    return try await shared.skipUnlessSupported(file: file, line: line) {
+      #if os(macOS)
+      guard let swift = await ToolchainRegistry.forTesting.default?.swift else {
+        throw NoSwiftInToolchain()
+      }
+
+      let project = try await SwiftPMTestProject(
+        files: [
+          "Lib/MyFile.swift": """
+          public func foo() {}
+          """,
+        ],
+        manifest: """
+        let package = Package(
+          name: "MyLibrary",
+          targets: [
+           .target(name: "Lib"),
+          ]
+        )
+        """
+      )
+      do {
+        var arguments = [
+          try swift.filePath, "build", "--package-path", try project.scratchDirectory.filePath, "--target", "Lib",
+          "--swift-sdk", "arm64-apple-ios",
+        ]
+        if let globalModuleCache = try globalModuleCache {
+          arguments += ["-Xswiftc", "-module-cache-path", "-Xswiftc", try globalModuleCache.filePath]
+        }
+        let status = try await Process.run(arguments: arguments, workingDirectory: nil)
+        guard case .terminated(code: 0) = status.exitStatus else {
+          let error = (try? String(decoding: status.stderrOutput.get(), as: UTF8.self)) ?? "unknown error"
+          return .featureUnsupported(skipMessage: "Cannot build for iOS: \(error)")
+        }
```

Comment:
Could this be the following?

```suggestion
        try await SwiftPMTestProject.build(at: try project.scratchDirectory, extraArguments: ["--swift-sdk", "arm64-apple-ios"])
```